### PR TITLE
feat(teritori): Update Assetlist with IBC denoms

### PR DESCRIPTION
**Pull request by omniwired**

### CODE REVIEW

File: teritori/assetlist.json [hunk 1]
```diff
@@ -29,6 +29,173 @@
...
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "cosmoshub",
+            "base_denom": "uatom",
+            "channel_id": "channel-10"
+          },
+          "chain": {
+            "channel_id": "channel-431",
+            "path": "transfer/channel-431/uatom"
```
Comment (JeremyParish69) [inline teritori/assetlist.json:109]: For all these assets I think you put the channel_id with the opposite chain/counterparty channel_id property. I think cosmoshub/counterparty would be channel-431 and chain channel_id would be channel-10. The path also doesn't yield the ibc/hash denom and swapping the channel order would fix this.

File: teritori/assetlist.json [hunk 2]
```diff
@@ -29,6 +29,173 @@
...
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "juno",
+            "base_denom": "ujuno",
+            "channel_id": "channel-11"
+          },
+          "chain": {
+            "channel_id": "channel-164",
```
Comment (JeremyParish69) [inline teritori/assetlist.json:140]: Adding this requires there to be a juno-teritori ibc connection registered.

File: teritori/assetlist.json [hunk 3]
```diff
@@ -29,6 +29,173 @@
...
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.svg"
+      },
+      "coingecko_id": "osmosis",
+      "keywords": [
+        "dex",
+        "staking"
+      ],
+      "images": [
+        {
```
Comment (JeremyParish69) [inline teritori/assetlist.json:74]: Please add an image_sync here (e.g., https://github.com/cosmos/chain-registry/blob/e1a5d27659072d77a5f5ecca42ea3c7969bd3a37/dyson/assetlist.json#L65)

**Comment by omniwired** [comment]
@JeremyParish69 thank you always for your review, I made the requested changes

**Pull request merged**